build/scripts/version-update/after-release/after-release.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("Please provide the initial version as a command-line argument")
+		return
+	}
+
+	initialVersion := os.Args[1]
+	files := []string{
+		"install/helm/agones/Chart.yaml",
+		"install/yaml/install.yaml",
+		"install/helm/agones/values.yaml",
+		"sdks/nodejs/package.json",
+		"sdks/nodejs/package-lock.json",
+		"sdks/unity/package.json",
+		"sdks/csharp/sdk/AgonesSDK.nuspec",
+		"sdks/csharp/sdk/csharp-sdk.csproj",
+	}
+
+	for _, filename := range files {
+		err := UpdateVersionInFile(filename, initialVersion)
+		if err != nil {
+			fmt.Printf("Error updating file %s: %s\n", filename, err.Error())
+		}
+	}
+}
+
+func UpdateVersionInFile(filename string, initialVersion string) error {
+	fileBytes, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+
+	content := string(fileBytes)
+
+	// Check the file extension and update version accordingly
+	ext := filepath.Ext(filename)
+	switch ext {
+	case ".yaml", ".yml", ".nuspec", ".csproj":
```

Comment:
I'm not sure this needs a case statement. It seems like you can just write it as

```
if ext := filepath.Ext(filename); ext == ".json" {
  updateJSONVersion()
} else {
  updateVersion()
}
```

Reply:
I tried to update the JSON files without a switch/case statement, but the syntax was slightly different. So created a switch/case statement to ensure that the files were updated correctly.